server/files/stylesheets/docs.css

```diff
@@ -2302,6 +2302,13 @@ code .chunk {
   font-size: 1.2em;
 }
 
+div.footer-sponsor {
```

Comment:
Let's skip the top margin for the upper most sponsor, it is not aligned with the social links to the left otherwise
```suggestion
div.footer-sponsor:not(:first-child) {
```
### Before
![image](https://user-images.githubusercontent.com/18379884/82117316-5ddab600-976f-11ea-9f9f-08fa2b906842.png)


### After
![image](https://user-images.githubusercontent.com/18379884/82117266-40a5e780-976f-11ea-8157-808ff9018a35.png)